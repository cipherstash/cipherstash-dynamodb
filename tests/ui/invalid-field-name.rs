use cipherstash_dynamodb::Encryptable;

#[derive(Debug, Encryptable)]
struct User {
    #[cipherstash(query = "exact")]
    #[partition_key]
    email: String,
    __name: String
}

fn main() {}
