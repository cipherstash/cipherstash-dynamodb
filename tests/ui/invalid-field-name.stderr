error: Invalid field '__name': fields must not be prefixed with __
 --> tests/ui/invalid-field-name.rs:8:5
  |
8 |     __name: String
  |     ^^^^^^^^^^^^^^
